spec/feature/registration_spec.rb

```diff
@@ -44,6 +44,19 @@
     assert_enqueued_jobs 1, only: NotifyDeliveryJob
   end
 
+  it "sends shows an account created security event" do
```

Comment:
Should be `it "shows an account created security event" do`

Reply:
fixed that here: https://github.com/alphagov/govuk-account-manager-prototype/pull/571/commits/7b34b4bd4145c0578bfa7ba3daf05fd4c69c9977